torch/csrc/jit/graph_executor.cpp

```diff
@@ -294,6 +295,7 @@ struct GraphExecutorImpl {
     auto local_graph = this->graph;
     if(all_dynamic(local_graph->inputs()) && all_dynamic(local_graph->outputs())) {
       local_graph = this->graph->copy();
+      EraseNumberTypes(local_graph);
```

Comment:
Is this here because PropagateInputShapes does not understand number types? Can we add a comment if so?

Reply:
Yes